.github/workflows/localization-update.yml

```diff
@@ -0,0 +1,24 @@
+name: Get Localization Translations
+on:
+  push:
+    branches:
+    - "lego/*"
+
+jobs:
+  pull-request:
+    name: [Open PR to main]
```

Comment:
We might use use another action like this in the future. Can we rename the action to something more localisation specific?